app/controllers/schemes_controller.rb

```diff
@@ -143,9 +143,9 @@ def next_page_path(page)
     when "support"
       new_location_path
     when "details"
-      if @scheme.support_services_provider_before_type_cast&.zero?
+      if @scheme.arrangement_type == "The same organisation that owns the housing stock"
         scheme_primary_client_group_path(@scheme)
-      elsif @scheme.support_services_provider_before_type_cast.positive?
```

Comment:
For conditionals I wouldn't rely on the text value (it can change based on design), but used the letter instead.